### PR TITLE
Nav awesome book--3: a complete website with navigation

### PR DESCRIPTION
**Pull request by omarbabou**
**In order to fulfill this milestone, I went through the following:**

      > Modified the Awesome books application markup to have:
            - A Navigation bar.
            - Three content sections:
               1. Books list.
               2. Add book form.
               3. Contact info.
            - A footer.
      > Styled our application with reference to the wireframe provided.
      > Updated our javascript code to make sure that only the section whose corresponding link in the nav-bar is clicked is 
        displayed at a time.
      > We also added the time and date stamp as was indicated in the wireframe.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,16 +14,53 @@
...
   </head>
   <body>
-    <h1>Awesome <span>Book</span>Store</h1>
-    <div class="displayBook"></div>
+    <header>
+      <h1>Awesome <span>Book</span>Store</h1>
+      <nav>
+        <ul>
+          <li class="list">List</li>
+          <li class="add">Add Contact</li>
```
Comment (phelian23) [inline index.html]: - [x] While the app is working as it should, it is important that the `nav links` is named appropriately according to the functions they are to perform. Kindly change this to `Add Book` or `Add new book`.

File: scripts/main.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+};
+
+navLink.forEach((link) => {
+  link.addEventListener('click', () => {
+    if (link.classList.contains('add')) {
+      displayWindow(link);
+      const newContact = document.querySelector('.form');
+      home.style.display = 'none';
+      newContact.classList.add('active');
+      document.querySelector('h2').textContent = 'Add new Contact';
```
Comment (phelian23) [inline scripts/main.js]: - [x] Also here, kindly change the text content to add new book

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @omarbabou,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
General Requirements 👌
Good commit messages 🙌
Design is outstanding 🚀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -14,16 +14,53 @@
...
-    <h1>Awesome <span>Book</span>Store</h1>
-    <div class="displayBook"></div>
+    <header>
+      <h1>Awesome <span>Book</span>Store</h1>
+      <nav>
+        <ul>
+          <li class="list">List</li>
+          <li class="add-new-book">Add Contact</li>
+          <li class="contact">Contact</li>
+        </ul>
```
Comment (codepantha) [inline index.html]: - [x] Like the previous reviewer suggested, please ensure that the links do exactly what they say. In this case, please change **Add Contact** to **Add Book**

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUESTED CHANGES ♻️ 

## Hi @omarbabou 👋, 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



### Highlights 🎯 
✔️ Descriptive PR
✔️ Beautiful UI design
✔️ Linter checks
✔️ Good commit history

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @omarbabou, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- You have done a great job working on the requested changes 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**